src/shared/utils/sqs.ts

```diff
@@ -9,8 +9,6 @@ export async function sendRecord(
   queueUrl: string,
   messageBody: string
 ): Promise<void> {
-  console.log("sending messageBody " + messageBody);
-
```

Comment:
Can we log just messageBody.event_id?

Reply:
one of the int test checks event id in the logs. For debugging purpose can we log eventid ?

Reply:
yup, done. I've added a condition around it because there's no reason why this should know about events. In theory you could use this function to put anything in a queue. A better solution would be to pass in a list of fields you'd like to log but that seemed like too much work for this PR